### PR TITLE
fix(bctheme): BCTHEME-18 Cornerstone - Text logos truncated when view…

### PR DESCRIPTION
**Pull request by BC-tymurbiedukhin**
…ed on mobile devices and smaller browser sizes

#### What?

Using Cornerstone, text logos are being cut off on mobile devices and smaller window sizes. 
Now logo is responsive due to depending on vw.

#### Tickets / Documentation

https://jira.bigcommerce.com/browse/BCTHEME-18

#### Screenshots (if appropriate)

responsive-text-logo.mov video is attached to the ticket

ping @golcinho @yurytut1993 @bc-alexsaiannyi @junedkazi

### CODE REVIEW

File: assets/scss/layouts/header/_header.scss [hunk 1]
```diff
@@ -65,11 +68,14 @@
     > a {
         @include clearfix;
         color: $storeName-color;
-        display: inline-block;
-        padding: $header-logo-paddingVertical 0;
+        height: inherit;
+        display: flex;
+        align-items: center;
+        margin-right: auto;
+        margin-left: auto;
```
Comment (yurytut1993) [inline assets/scss/layouts/header/_header.scss:75]: can we replace it to
`margin: 0 auto;` ?

File: assets/scss/layouts/header/_header.scss [hunk 2]
```diff
@@ -65,11 +68,14 @@
     > a {
```
Comment (yurytut1993) [inline assets/scss/layouts/header/_header.scss:68]: Can we replace tag selector on class selector? I know that is not part of your ticket, but if it is fast - will be great

Comment (BC-tymurbiedukhin) [reply assets/scss/layouts/header/_header.scss]: Done

File: assets/scss/layouts/header/_header.scss (hunk 1 above)
Comment (BC-tymurbiedukhin) [reply assets/scss/layouts/header/_header.scss]: Done

**Pull request merged**